datastore/query.go

```diff
@@ -176,6 +323,22 @@ func (q *Query) Transaction(t *Transaction) *Query {
 	return q
 }
 
+// FilterEntity returns a query with provided filter.
+//
+// Filter can be a single field comparison or a composite filter
+// AND and OR are supported composite filters
+// Filters in multiple calls are joined together by AND
+func (q *Query) FilterEntity(entityFilter EntityFilter) *Query {
```

Comment:
nit: the type already tells us it's a "EntityFilter" so the argument name could just be `e` or `ef`.

Reply:
Done